meteor/client/ui/RundownView/RundownTiming.tsx

```diff
@@ -251,7 +251,7 @@ export const RundownTimingProvider = withTracker<
 			}
 			if (prevProps.parts !== this.props.parts) {
 				// empty the temporary Part Instances cache
-				this.temporaryPartInstances = {}
+				this.onRefreshTimer()
```

Comment:
```this.temporaryPartInstances = {}```
Should not be removed. The temporaryPartInstances is another speed-up/improvement that has been implemented at NRK and should be kept.

Reply:
Both lines are now included.